_themes/rtd-blog/social.html

```diff
@@ -56,10 +56,13 @@ <h3> Follow us </h3>
 
 <ul>
 <li>
-<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">Atom Feed</a>
+<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">Atom Feed 📶️</a>
 </li>
 <li>
-<a href="https://twitter.com/readthedocs" rel="me">Twitter</a>
+<a href="https://fosstodon.org/@readthedocs/" rel="me">Mastodon 🐘️</a>
```

Comment:
```suggestion
<a href="https://fosstodon.org/@readthedocs" rel="me">Mastodon 🐘️</a>
```

Reply:
Ah yes, without the slash is normal. I must have hit some kind of random error, I tried adding the slash and then the error went away :shrug: